osc/commandline.py

```diff
@@ -4259,7 +4259,13 @@ def do_maintenancerequest(self, subcmd, opts, *args):
             print(f'Using target project \'{target_project}\'{release_in}')
 
         if not opts.message:
-            opts.message = edit_message()
+            if opts.supersede:
+                from .obs_api import Request
+
+                req = Request.from_api(apiurl, opts.supersede)
+                msg = req.description + "\n"
+            opts.message = edit_message(template=f"{msg}")
```

Comment:
The scanner revealed a problem with undefined variable.
How about adding `msg = ""`?

```
            msg = ""

            if opts.supersede:
                from .obs_api import Request

                req = Request.from_api(apiurl, opts.supersede)
                msg = req.description + "\n"
	
            opts.message = edit_message(template=f"{msg}")
```

Reply:
Yeah, I saw this as well. 
This is what happens when implementing this in the in place for easy testing it and then cut-n-paste code over into the git sources. 
Meanwhile I've pushed a fix. Sorry for the noise.